.drone.yml

```diff
@@ -20,6 +20,11 @@ steps:
     volumes:
       - name: docker
         path: /var/run/docker.sock
+    environment:
```

Comment:
I will note that we don't usually give credentials to PR CI, only Merge. Is that OK?

Reply:
Good point, I forgot about that. I removed the credentials part, we should be good with the rate limit as we don't run it that often.